calico-cloud/image-assurance/exceptions.mdx

```diff
@@ -0,0 +1,69 @@
+:::
+
+## Exclude a vulnerability
+
+To create a vulnerability exception:
+1. Go to **Image Assurance**, **All Scanned Images**.
+2. Select an image from the list.
+3. On the panel, open the Package, and click **Actions**, **Add Exception**.
+3. Enter the required information and click **Save**. The exception is immediately applied to your scan results.
+
+## Exclude many vulnerabilities
+
+To create many vulnerability exceptions using a CSV file, follow these steps:
+
+1. Go to **Image Assurance**, **All Scanned Images**.
+2. Click the **Export** button and select **Export data**.
+3. Select **CSV** as the file type and **Export a row for each image and CVE IDs** as the table style.
+4. Open the exported CSV file in your preferred editor.
+5. Add two columns: **Justification** and **Scope**.
+
+The **Scope** column can have three values:
+
+- **any**: The exception applies to all images.
+- **image**: The exception applies to all versions of the image.
+- **image_tags**: The exception applies to specific tags of an image.
+
+Required CSV headers and their explanation:
+
+| Header        | Explanation                                       | Required By The **any** Scope | Required By The **image** Scope | Required By The **image_tags** Scope |
+|---------------|---------------------------------------------------|-------------------------------|---------------------------------|--------------------------------------|
+| CVE           | The name of the CVE (e.g., CVE-2021-44228)        | Yes                           | Yes                             | Yes                                  |
+| Registry      | Where the collection of images is stored          | No                            | Yes                             | Yes                                  |
+| Repository    | The name of the image                             | No                            | Yes                             | Yes                                  |
+| Tags          | A JSON list of image tags                         | No                            | No                              | Yes                                  |
+| Justification | Explanation for creating the exception            | Yes                           | Yes                             | Yes                                  |
+| Scope         | How the exception is applied                      | Yes                           | Yes                             | Yes                                  |
+
+Example CSV:
+
+```
+CVE,Registry,Repository,Tags,Justification,Scope
+CVE-2024-1234,mycontainerregistry.io/my-org,my-application,"[""v3.4.5""]",justification,image_tags
+CVE-2024-2345,mycontainerregistry.io/my-org,my-application,"[""v1.2.3"",""v2.3.4""]",justification,image_tags
+CVE-2024-3456,mycontainerregistry.io/my-org,my-application,,justification,image
+CVE-2024-4567,,,,justification,any
+```
+
+Upload the formatted CSV:
+
+1. TODO write this
```

Comment:
@stevegaossou , there is a TODO here. My suggestion:

1. Go to **Image Assurance**, **Vulnerability Exceptions**.
2. Click on the top-right arrow-shaped.
3. Follow the instructions provided in the pop-up modal to create the vulnerability exceptions.